docs/advanced-features/custom-document.md

```diff
@@ -43,17 +43,18 @@ Custom attributes are allowed as props, like `lang`:
 <Html lang="en">
 ```
 
+The `<Head />` component used here is not the same one from [`next/head`](/docs/api-reference/next/head), to avoid duplicated tags, have a custom `<title>`, or to add dynamic tags, like meta tags, use [`next/head`](/docs/api-reference/next/head) instead.
```

Comment:
Sorry, the second half of this sentence didn’t make much sense to me. Maybe:

```suggestion
The `<Head />` component used here is not the same one from [`next/head`](/docs/api-reference/next/head). The `<Head />` component used here should only be used for any `<head>` code that is common for all pages. For all other cases, we recommend using [`next/head`](/docs/api-reference/next/head) in your pages or components.
```

Reply:
@chibicode Can we make it mention both? it's important that we mention that there are no dynamic tags, that we don't remove duplicated tags (because `next/head` does), and that the `<title>` can cause conflicts